pkg/utils/helm/utils.go

```diff
@@ -64,9 +64,10 @@ func InstallRelease(name string, namespace string, valueFile string, chartName s
 	return err
 }
 
-/**
-* check if the release exist
- */
+
+// CheckRelease checks if the release derived from given name and namespace exist.
```

Comment:
`derived from` looks a little bit weird to me
```suggestion
// CheckRelease checks if the release with the given name and namespace exist.
```

Reply:
I have already modify "derived from" to "with the"